### ISSUE TITLE
Allow launching URL/App on button press

### ISSUE THREAD

**Issue by dotMorten** [cgiesche/streamdeck-homeassistant#172]
I'd love to be able to launch a browser or app via a button-press, but currently you can only set it do something on Home Assistant. For instance it might show my current solar production, but when pressing it, it should take me to my solar providers dashboard.

**Comment by Hessenpower01** [comment]
No reason to do this from HA, it's available in the standard Stream Deck functionality.

**Comment by dotMorten** [comment]
Yes as a separate button. But if I have a HA button that shows for instance current solar production and I press it, I can set up what that does - I want to be able to launch a browser to show the details
